### ISSUE TITLE
Ability to normalise volume

### ISSUE THREAD

**Issue by TheArcaneBrony** [ytdl-org/youtube-dl#27864]
## Checklist

- [x] I'm reporting a feature request
- [x] I've verified that I'm running youtube-dl version **2021.01.16**
- [x] I've searched the bugtracker for similar feature requests including closed ones

## Description

Ability to normalise volume on sites that provide this data (eg. YouTube), useful especially in playlists with music. When downloading a playlist currently, some songs are loud, and occasionally one will be quieter, needing you to turn up your volume, causing too high volume for following videos/songs.

**Comment by october262** [comment]
you can use ffmpeg to adjust the music volume - https://www.linuxquestions.org/questions/linux-software-2/large-number-of-mkv-mp4-webm-videos-%5Byoutube-dl%5D-need-to-adjust-volume-of-em-4175620725/

**Comment by TheArcaneBrony** [comment]
Yes i know, its just annoying when you need to do it for a 120 video playlist for example
